packages/repository-json-schema/src/build-schema.ts

```diff
@@ -148,6 +160,8 @@ export function modelToJsonSchema(ctor: Function): JSONSchema {
   }
 
   result.title = meta.title || ctor.name;
+  const isVisited =
+    options && options.visited && options.visited.includes(result.title!);
```

Comment:
Let's set `options` to `{}` by default so that we don't have to check for its existence.

```ts
export function modelToJsonSchema(
  ctor: Function,
  options: JsonSchemaOptions = {}
): JSONSchema {
  // ...

  const isVisited =
    options.visited && options.visited.includes(result.title!);

  // ...
}
```